src/test/java/com/hocs/test/pages/comp/COMPProgressCase.java

```diff
@@ -0,0 +1,101 @@
+package com.hocs.test.pages.comp;
```

Comment:
Although it does make things easier, does this not go against the structure of the pack?

Reply:
So WCS already had this structure, and I thought it makes progressing a case a lot clearer. I was updating COMP to use the recording helper methods anyway (so I could combine the data and workflow scenarios), so at the same time I changed COMP over to the WCS version of the structure. 
When I have the time I want to do the same for MPAM and DCU, but I wanted to submit this PR before it became something that changed the whole pack.